docs/Configuring-Orion/Using-PostgreSQL.md

```diff
@@ -0,0 +1,59 @@
+description: Storing Information with PostgreSQL 
+<!--- END of page meta data -->
+
+# Store Information with PostgreSQL
+
+PostgreSQL is an object-relational database you can use to store private transaction payloads for Orion. Consider using this option as an aid in [disaster recovery](Disaster-Recovery.md).
+
+## Prerequisites
+
+* [PostgreSQL](https://www.postgresql.org/)
+
+## Set up your PostgreSQL database
+
+In your database:
+
+1. Create the `store` table by using the provided DDL script, as shown in the following example. Substute your own values for `HOST-NAME`, `USER-NAME`, and `DATABASE-NAME`.
+
+    `psql -h <HOST-NAME> -U <USER-NAME> -d <DATABASE-NAME> -f ~/orion/docs/configuring-orion/database/postgres_storage.sql` 
```

Comment:
The path to `progres_storage.sql` will need to be updated to show it'll have to be replaced to the path the script (ie, the path you've given assumes that you're using the source code doesn't it?)